sky/services/platform/interface.mojom

```diff
@@ -0,0 +1,95 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module platform_interface;
+
+/// Specifies an acceptable orientation for the interface to be in
+/// A mask of acceptable interface orientations is created by or'ing individual
+/// options
+enum /* options */ InterfaceOrientation {
+  Portrait  = 1,
+  Landscape = 2,
+};
+
+/// Specifies a particular device orientation
+/// A mask of device interface orientations is created by or'ing individual
+/// options
+enum /* options*/ DeviceOrientation {
```

Comment:
Good idea, its best to ambiguity w.r.t this thing. Especially when dealing with all the different variants on phones and tablets